File: Assignment_9/program1.c
#include<stdio.h>

void DisplayDigit(int iNo)
{
    if(iNo < 0)
    {
        iNo = -iNo;
    }
    int iDigit = 0;

    while (iNo != 0)
    {
        iDigit = iNo % 10;
        iNo = iNo / 10;
        printf("%d\n",iDigit);

    }
}

int main()
{

    int iValue  = 0;

    printf("Enter the value : ");
    scanf("%d",&iValue);

    DisplayDigit(iValue);

    return 0;
}

File: Assignment_6/program1.java
import java.util.Scanner;


class Check
{

    public void CheckNumber(int iNo)
    {
        if(iNo < 50)
        {
            System.out.println("Small");
        }
        else if((iNo > 50) && (iNo < 100))
        {
            System.out.println("Medium");
        }
        else if (iNo > 100)
        {
            System.out.println("Large");
        }


    }
} 

class program1
{
    public static void main(String A[])
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        Check cobj = new Check();

        cobj.CheckNumber(iValue);


    }
}